include/boost/geometry/algorithms/detail/overlay/get_turns.hpp

```diff
@@ -187,11 +187,8 @@ private :
         // So advance to the "non duplicate next"
         // (the check is defensive, to avoid endless loops)
         std::size_t check = 0;
-        while(! detail::disjoint::disjoint_point_point
-                (
-                    current_robust_point, next_robust_point,
-                    disjoint_strategy_type()
-                )
+        while(! detail::disjoint::disjoint_point_point(
```

Comment:
a space is missing after `while`

Reply:
Indeed, a leftover from the previous version. I'll fix it.